doc/nrf/ug_partition_manager.rst

```diff
@@ -279,3 +279,38 @@ CMake usage
 
       --slot-size $<TARGET_PROPERTY:partition_manager,MCUBOOT_SLOT_SIZE>
 
+
+Predefined configuration
+========================
+Predefined configuration is a feature intended for deployed products which consists of multiple images, but where
```

Comment:
```suggestion
The predefined configuration is a feature intended for deployed products which consist of multiple images, but where
only a subset of the included images will be upgradeable through firmware update mechanisms.
In these use cases, it is crucial that the images to be upgraded are linked to the same address as what is on the existing product.
This to avoid accidental overrides and incorrect configurations.
Within the upgradeable area, the predefined configuration also allows defining new partitions, deleting existing partitions, and modifying existing partitions.
```